nixos/modules/services/web-apps/freshrss.nix

```diff
@@ -243,6 +244,9 @@ in
         in
         {
           description = "Set up the state directory for FreshRSS before use";
+          wants = [ "network-online.target" ];
+          after = [ "network-online.target" ];
```

Comment:
It seems `systemd.services.freshrss-config` does not need network connection (I am not sure though).  If so, these two lines of `network-oneline.target` should be moved to the `systemd.services.freshrss-updater`.

Reply:
In my testing the systemd unit fails if no network is available, hence why I added these lines.